alchemist/alchemist-incarnation-biochemistry/src/main/java/it/unibo/alchemist/model/implementations/actions/ChemiotacticPolarization.java

```diff
@@ -0,0 +1,117 @@
+package it.unibo.alchemist.model.implementations.actions;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.math3.util.FastMath;
+
+import it.unibo.alchemist.model.implementations.molecules.Biomolecule;
+import it.unibo.alchemist.model.implementations.positions.Continuous2DEuclidean;
+import it.unibo.alchemist.model.interfaces.CellNode;
+import it.unibo.alchemist.model.interfaces.Context;
+import it.unibo.alchemist.model.interfaces.Environment;
+import it.unibo.alchemist.model.interfaces.EnvironmentNode;
+import it.unibo.alchemist.model.interfaces.Node;
+import it.unibo.alchemist.model.interfaces.Position;
+import it.unibo.alchemist.model.interfaces.Reaction;
+
+/**
+ *
+ */
+public class ChemiotacticPolarization extends AbstractAction<Double> {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = -6147634743476443019L;
+    private final Environment<Double> env;
+    private final Biomolecule biomol;
+    private final boolean ascend;
+
+    /**
+     * Initialize a polarization activity regulated by environmental concentration of a molecule.
+     * @param environment 
+     * @param node 
+     * @param biomol 
+     * @param ascendGrad if that parameter is true, the polarization versor of the cell will be directed in direction of the greates concentration of biomolecule in neighborhood; if it's false, the versor will be directed in the exactly the opposite direction.
+     */
+    public ChemiotacticPolarization(final Environment<Double> environment, final Node<Double> node, final String biomol, final String ascendGrad) {
+        super(node);
+        if (!(node instanceof CellNode)) {
+            throw  new UnsupportedOperationException("Polarization can happen only in cells.");
+        }
+        this.env = environment;
+        this.biomol = new Biomolecule(biomol);
+        if (ascendGrad.equals("up")) {
+            this.ascend = true;
+        } else if (ascendGrad.equals("down")) {
+            this.ascend = false;
+        } else {
+            throw new IllegalArgumentException("Possible imput string are only up or down");
+        }
+    }
```

Comment:
Add another constructor that requires a Biomolecule rather than a String, than call it (improves the API).